### PR TITLE
tikametadata rule check

### PR DESCRIPTION
**Pull request by orhlulu**
this is for tika metadata rule check

### CODE REVIEW

File: src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  static final String LABEL = "Tika Meta Regex";
+  static final RuleType RULE_TYPE = RuleType.TIKA_METADATA;
+  static final List<Parameter> PARAMETERS =
+      Lists.newArrayList(new Parameter("headers", "aa,bb,cc", ParamType.STRING_LIST));
+
+  public TikaMetaValue() {
+    this.setLabel(LABEL);
+    this.setRuleType(RULE_TYPE);
+    this.setParams(PARAMETERS);
```
Comment (yclim) [inline src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java:27]: the default constructor should take 2 string parameter. call `this(null, null)` instead

File: src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+public class TikaMetaValue extends Rule {
+
+  static final String LABEL = "Tika Meta Regex";
+  static final RuleType RULE_TYPE = RuleType.TIKA_METADATA;
+  static final List<Parameter> PARAMETERS =
+      Lists.newArrayList(new Parameter("headers", "aa,bb,cc", ParamType.STRING_LIST));
+
+  public TikaMetaValue() {
+    this.setLabel(LABEL);
```
Comment (yclim) [inline src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java:25]: the default constructor should take 2 string parameter. call `this(null, null)` instead

File: src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    String keyMatch = "";
+    String valueMatch = "";
+
+    for (Parameter param : getParams()) {
+      if(param.getLabel().equals("key")) {
+        keyMatch = param.getValue();
+      }else if (param.getLabel().equals("value")) {
+        valueMatch = param.getValue();
+      }
+
```
Comment (yclim) [inline src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java:69]: remove comment

File: src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java (hunk 2 above)
Comment (orhlulu) [reply src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java]: ok done

**Pull request merged**